uaclient/cli/vulnerability/list.py

```diff
@@ -0,0 +1,460 @@
+from typing import Dict, NamedTuple
+
+from uaclient import config, exceptions, messages
+from uaclient.api.u.pro.security.vulnerabilities.cve.v1 import (
+    CVEVulnerabilitiesOptions,
+)
+from uaclient.api.u.pro.security.vulnerabilities.cve.v1 import (
+    _vulnerabilities_with_applied_fixes_count as cve_vulnerabilities,
+)
+from uaclient.api.u.pro.security.vulnerabilities.usn.v1 import (
+    USNVulnerabilitiesOptions,
+)
+from uaclient.api.u.pro.security.vulnerabilities.usn.v1 import (
+    _vulnerabilities_with_applied_fixes_count as usn_vulnerabilities,
+)
+from uaclient.cli.commands import ProArgument, ProArgumentGroup, ProCommand
+from uaclient.cli.formatter import Table
+from uaclient.cli.vulnerability import util as vuln_util
+
+VulnerabilityCountInfo = NamedTuple(
+    "VulnerabilityCountInfo",
+    [("count", int), ("count_by_priority", Dict[str, int])],
+)
+
+
+def _create_usn_table(usns, num_rows=None):
+    usns_sorted_by_name = sorted(usns, key=lambda usn: usn.name)
+
+    if num_rows:
+        usns_sorted_by_name = usns_sorted_by_name[:num_rows]
+
+    return Table(
+        headers=[
+            "VULNERABILITY",
+            "FIX AVAILABLE FROM",
+            "AFFECTED INSTALLED PACKAGES",
+        ],
+        rows=[
+            [
+                usn.name,
```

Comment:
For both USNs and CVEs, we want this to be a link to our securty page/docs where we can find more information about CVEs.

Reply:
FYI I added links in #3327
If this gets merged before that one, we can add the links later